File: benchmark/2026-09-17-7677581(latest)/semantic-compression/probes/go/F04.P1.go
package main

import "fmt"

func probe() int32 {
	// BEGIN PROBE F04.P1
	var slot int32 = 4
	port := &slot
	*port = 9
	return slot
	// END PROBE F04.P1
}

func main() {
	fmt.Println(probe())
}

File: benchmark/2026-09-17-7677581(latest)/semantic-compression/probes/cpp/F04.P1.cpp
#include <cstdint>
#include <cstdio>

std::int32_t probe() {
// BEGIN PROBE F04.P1
std::int32_t slot = 4;
auto& port = slot;
port = 9;
return slot;
// END PROBE F04.P1
}

int main() {
    std::printf("%d\n", probe());
}
